ui/app/SheetStorage.js

```diff
@@ -233,6 +243,32 @@ export class SheetStorage {
   }
 }
 
+class Config {
+  constructor(path, defaultValue = '') {
+    this.defaultValue = defaultValue;
+    this.path = path;
+    this.signal = signal(defaultValue);
```

Comment:
What do signals do?

Reply:
They're variables which trigger re-rendering on change.

They seem like they might be an improvement over passing state all over the place.

I'm hoping that they allow much of the logic to become decentralized.